package.json

```diff
@@ -40,31 +40,31 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^4.24.0"
+    "@typescript-eslint/experimental-utils": "^4.26.0"
```

Comment:
I made sure we use the newest Jest version and updated all other dependencies too

Reply:
There is no problem updating dev dependencies that often. What I meant is to avoid doing so for production updates. Then this PR shouldn't update `@typescript-eslint/experimental-utils` to avoid releasing another version of the plugin by reasons mentioned in the original comment.